scripts/install/install_bmi.sh

```diff
@@ -0,0 +1,58 @@
+set -x
+
+### START BMI setup
+
+TOP_DIR=$(cd $(dirname "$0") && pwd)
+
+# Move to main ims directory for install from script location
+pushd $TOP_DIR/../../
+
+### Install BMI
+
+user=`whoami`
+sudo mkdir -p /var/log/bmi/logs /etc/bmi /opt/bmi /var/lib/tftpboot
+sudo chown -R $user:$user /etc/bmi /var/log/bmi /opt/bmi /var/lib/tftpboot /etc/tgt
+
+### Copy config/template files
+cp bmi_config.cfg.test /etc/bmi/bmiconfig.cfg
+cp ims/ipxe.temp /etc/bmi/ipxe.cfg
+cp ims/mac.temp /etc/bmi/pxelinux.cfg
+
+virtualenv .bmi_venv
+source .bmi_venv/bin/activate
+pip install python-cephlibs
+python setup.py develop
+
+touch /opt/bmi/bmi.db
+
+export HAAS_USERNAME=haas
+export HAAS_PASSWORD=secret
+
+bmi db ls
+sqlite3 /opt/bmi/bmi.db "insert into project values (0, 'bmi_infra', 'bmi_network')"
+
+# Temporary changes to get stable CI. Need to do pull request to get this fixed.
+sed -i s/"HAAS_BMI_CHANNEL = \"vlan\/native\""/"HAAS_BMI_CHANNEL = \"null\""/ ims/common/constants.py
```

Comment:
My only concern is people might accidently commit it. Since we have an alternative